tests/src/e2e/eth_bridge_tests.rs

```diff
@@ -237,3 +240,69 @@ fn test_add_to_bridge_pool() {
     .unwrap();
     namadar.exp_string(r#""bridge_pool_contents":"#).unwrap();
 }
+
+/// Tests that the ledger configures its Ethereum oracle with values from
+/// storage, if the Ethereum bridge has been bootstrapped for the Namada chain.
+#[test]
+fn test_configure_oracle_from_storage() -> Result<()> {
+    let ethereum_bridge_params = EthereumBridgeConfig {
+        min_confirmations: MinimumConfirmations::from(unsafe {
+            // SAFETY: The only way the API contract of `NonZeroU64` can
+            // be violated is if we construct values
+            // of this type using 0 as argument.
+            NonZeroU64::new_unchecked(10)
+        }),
+        contracts: Contracts {
+            native_erc20: EthAddress([1; 20]),
+            bridge: UpgradeableContract {
+                address: EthAddress([2; 20]),
+                version: ContractVersion::default(),
+            },
+            governance: UpgradeableContract {
+                address: EthAddress([3; 20]),
+                version: ContractVersion::default(),
+            },
+        },
+    };
+
+    // use a network-config.toml with eth bridge parameters in it
+    let test = setup::network(
+        |mut genesis| {
+            genesis.ethereum_bridge_params = Some(ethereum_bridge_params);
+            genesis
+        },
+        None,
+    )?;
+
+    // start the ledger with the real oracle and wait for a block to be
+    // committed
+
+    // TODO(namada#1061): need to start up a fake Ethereum node here for the
+    // oracle to connect to, to avoid errors in the ledger logs
```

Comment:
At the moment, the `namadan` logs gets spammed with error messages about the oracle not being able to connect to an Ethereum RPC endpoint, for this test it doesn't matter as we are just testing the oracle is initially configured correctly, but when the linked issue [namada#1061](https://github.com/anoma/namada/issues/1061) is done we can change this here also)